wiki/Help_Centre/Beatmapping/en.md

```diff
@@ -0,0 +1,67 @@
+# Beatmapping (help page)
+
+*For a list of all Help Centre articles, see: [Help Centre](/wiki/Help_Centre)*
+
+## How do I upload/update my map?
+
+**You can do this through the in-game editor.**
+
+To upload your beatmap, go into the editor and hit `File` > `Upload Beatmap`.
+
+This will make your beatmap available for others to download on the website and create a dedicated thread for discussion. Updating a map is the same process for a map that has already been uploaded.
+
+Each user has a limited number of maps they can simultaneously upload. Leaving a map inactive for a month (by not updating it or not posting in its thread) will send it to the Beatmap Graveyard. To update a graveyarded beatmap, you'll need to revive it first.
+
+## The editor is telling me that the beatmap I'm trying to submit isn't mine!
+
+**This can happen if the creator of the beatmap is not properly set within the beatmap files.**
+
+In the editor, click `File` in the menu, then `Open .osu file in Notepad`. Check the line `Creator:` and make sure that it's set to your exact username, capitalization included.
+
+If it isn't, manually change it for every difficulty in your set, reload the map, and you should be able to submit it.
+
+### I'm an osu!supporter, but I don't have 10 pending slots!
+
+**The number of slots available depends on the number of Ranked maps you have.**
+
+By default, every user has **2 pending slots**. 1 slot is added for every Ranked map the user owns, up to a total of 8 slots. Being a supporter grants 2 additional slots, from 4 (without Ranked maps), up to a total of 10.
+
+### How can I delete my map?
+
+**The osu! team does not take requests to manually delete uploaded beatmaps.**
+
+You can delete your beatmap yourself by waiting for it to decay to the beatmap Graveyard, which it will do so after a month without updates.
+
+Once your map is in the Graveyard, you can delete it yourself by going to it's beatmap page, clicking the `Discussion` button (which will bring you to its forum page), and then clicking the `Delete` button that appears next to the beatmap link.
```

Comment:
```suggestion
Once your map is in the Graveyard, you can delete it yourself by going to it's beatmap page, clicking the `Discussion` button (which will bring you to its modding discussion page), and then clicking the `Delete` button that appears next to the beatmap link.
```
I wonder if the part in parenthesis is actually needed at all? 

Also a bit unsure about the last part of the sentence. It's true that there is a Beatmap Page button right next to the delete button, but I feel like there may be a better way of describing where the delete button is.

Reply:
I think just knowing that the delete button is on that page and not hidden behind any dropdown menus is enough for most people. If it's really that vague we can add an image or something.